docs/send-data/sumo-logic-distribution-for-opentelemetry-collector/configuration.md

```diff
@@ -0,0 +1,779 @@
+---
+id: configuration
+title: OpenTelemetry Configuration
+---
+
+- [Basic configuration](#basic-configuration)
```

Comment:
Why is the table of contents here, but it has been removed from other files (known-issues.md, troubleshooting.md)?

Reply:
I'm still trying to figure out what to do

> 

Thank you, @astencel-sumo ! I'll let the S3M team know through Slack and loop you in.

Reply:
This file was so much longer than the others. I've removed it from here, but wanted to discuss with you all possibly splitting into different docs.